### PR TITLE
[Batch] Remove dependency and update live test case

### PR DESCRIPTION
**Pull request by xingwu1**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/latest/test_batch_data_plane_commands.py [hunk 1]
```diff
@@ -159,6 +159,7 @@ def test_batch_task_create_cmd(self, resource_group, batch_account_name):
             't_file': self._get_test_data_file('batchCreateTask.json'),
             'ts_file': self._get_test_data_file('batchCreateMultiTasks.json')
         })
+        is_playback = os.path.exists(self.recording_file)
```
Comment (troydai) [inline src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/latest/test_batch_data_plane_commands.py]: https://github.com/Azure/azure-cli/blob/dbe24401b163ac2bac4f993982375296414b81b7/src/command_modules/azure-cli-role/azure/cli/command_modules/role/tests/latest/test_role.py#L288

Comment (xingwu1) [reply src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/latest/test_batch_data_plane_commands.py]: @troydai Fixed.

**Comment by troydai** [comment]
Please fix the CI failures.

**Comment by xingwu1** [comment]
@troydai  @yugangw-msft  Could you sign off my PR?

**Comment by troydai** [comment]
Thank you.

**Pull request merged**